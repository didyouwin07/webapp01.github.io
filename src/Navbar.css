.navbar{
  display: flex;
  justify-content: space-between;
  padding: 33px 41px;
  flex-direction: column;
}

.logo{
  text-align: left;
  margin-bottom: 264px;
}

.buttons{
  display: flex;
  flex-direction: row;
  justify-content: flex-start;
  width: 80%;
  margin-top: 175px;
}

.ios-button{
  padding:13px;
}

.highlight-text{
  font-size: 42px;
  text-align: left;
}

.child-text{
  font-size: 16px;
  color: #6D6D6D;
  text-align: left;
}
