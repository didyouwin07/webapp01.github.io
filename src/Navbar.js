import "bootstrap/dist/css/bootstrap.min.css";
import "bootstrap/dist/js/bootstrap.bundle.min";
import MobileStoreButton from "react-mobile-store-button";
import "./Navbar.css";
import React from "react";

const Navbar = (): any => {
  return (
    <nav data-tesid="nav" className="navbar">
      <div className="logo">
        <img src="/Cat-N-Dog-Logo copy.png" />
      </div>
      <div className="highlight-text">
        <p>
          Lorem ipsum is simply dummy text of the printing and typesetting
          industry.
        </p>
      </div>
      <div className="child-text">
        Lorem ipsum has been the industry's standard dummy text ever since the
        1550s, when an unknown printer took a gallery of type and scrambled it
        to make a type specimen book.
      </div>
      <div className="buttons">
        <MobileStoreButton store="ios" url="/" className="ios-button" />
        <MobileStoreButton store="android" url="/" />
      </div>
    </nav>
  );
};

export default Navbar;
