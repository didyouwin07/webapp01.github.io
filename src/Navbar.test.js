import { render, screen } from "@testing-library/react";
import Navbar from "./Navbar";
import React from "react";
import ReactDOM from "react-dom";
import { cleanup } from "@testing-library/react";
import "@testing-library/jest-dom/extend-expect";
import renderer from "react-test-renderer";

afterEach(cleanup);

it("renders without crashing", () => {
  const div = document.createElement("div");
  ReactDOM.render(<Navbar></Navbar>, div);
});

it("renders button correctly", () => {
  const { getByTestId } = render(<Navbar></Navbar>);
});
it("matches snapshot", () => {
  const tree = renderer.create(<Navbar></Navbar>).toJSON();
  expect(tree).toMatchSnapshot();
});
