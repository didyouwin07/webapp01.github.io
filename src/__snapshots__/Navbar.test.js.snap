// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<nav
  className="navbar"
  data-tesid="nav"
>
  <div
    className="logo"
  >
    <img
      src="/Cat-N-Dog-Logo copy.png"
    />
  </div>
  <div
    className="highlight-text"
  >
    <p>
      Lorem ipsum is simply dummy text of the printing and typesetting industry.
    </p>
  </div>
  <div
    className="child-text"
  >
    Lorem ipsum has been the industry's standard dummy text ever since the 1550s, when an unknown printer took a gallery of type and scrambled it to make a type specimen book.
  </div>
  <div
    className="buttons"
  >
    <div
      className="ios-button"
      style={
        Object {
          "display": "inline-block",
          "height": 75,
          "width": 255,
        }
      }
    >
      <a
        href="/"
        rel="noopener noreferrer"
        style={
          Object {
            "background": "url(https://linkmaker.itunes.apple.com/images/badges/en-us/badge_appstore-lrg.svg) no-repeat",
            "backgroundSize": "contain",
            "display": "inline-block",
            "height": "100%",
            "overflow": "hidden",
            "padding": "5px",
            "textDecoration": "none",
            "width": "100%",
          }
        }
        target="_blank"
      >
         
      </a>
    </div>
    <div
      style={
        Object {
          "display": "inline-block",
          "height": 75,
          "width": 255,
        }
      }
    >
      <a
        href="/"
        rel="noopener noreferrer"
        style={
          Object {
            "background": "url(https://play.google.com/intl/en_us/badges/images/generic/en_badge_web_generic.png) no-repeat",
            "backgroundSize": "contain",
            "display": "inline-block",
            "height": "100%",
            "overflow": "hidden",
            "padding": "5px",
            "textDecoration": "none",
            "width": "100%",
          }
        }
        target="_blank"
      >
         
      </a>
    </div>
  </div>
</nav>
`;
